### PR TITLE
Support changing bundle type in packagers/optimizers

### PR DESCRIPTION
**Pull request by devongovett**
This adds support for changing the output file type in packagers/optimizers. This could be used for things like static rendering of markdown to HTML, and possibly other usecases.

### CODE REVIEW

File: packages/core/core/src/PackagerRunner.js [hunk 1]
```diff
@@ -428,18 +454,26 @@ export default class PackagerRunner {
...
-      bundle.filePath = filePath;
-      bundle.name = nullthrows(bundle.name).replace(
-        thisHashReference,
-        thisNameHash,
-      );
+      name = name.replace(thisHashReference, thisNameHash);
     }
 
+    bundle.filePath = filePath;
+    bundle.name = name;
```
Comment (devongovett) [inline packages/core/core/src/PackagerRunner.js:475]: Not sure about mutating the bundle here, but we were doing it before. 😬

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 6.87s | -92.00ms |
| Cached | 2.48s | +9.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 302.00ms | -31.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 301.00ms | -31.00ms 🚀 |
| dist/modern/kitchen-sink.90de237b.js | 1.16kb | +0.00b | 642.00ms | -1.16s 🚀 |
| dist/legacy/index.html | 709.00b | +0.00b | 1.67s | +911.00ms ⚠️ |
| dist/modern/index.html | 709.00b | +0.00b | 1.66s | +911.00ms ⚠️ |
| dist/modern/styles.cda8c91b.css | 78.00b | +0.00b | 1.53s | -135.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 283.00ms | -17.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 280.00ms | -20.00ms 🚀 |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 30.01s | -482.00ms |
| Cached | 2.64s | +4.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.89fe2d7f.js | 4.61kb | +0.00b | 12.63s | +6.21s ⚠️ |
| dist/UserProfile.8cce6c4b.js | 1.83kb | +0.00b | 12.63s | +6.21s ⚠️ |
| dist/NotFound.98e75551.js | 468.00b | +0.00b | 12.63s | +6.21s ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.42kb | +0.00b | 352.00ms | -69.00ms 🚀 |
| dist/PermalinkedComment.89fe2d7f.js | 4.61kb | +0.00b | 342.00ms | -77.00ms 🚀 |
| dist/UserProfile.8cce6c4b.js | 1.83kb | +0.00b | 342.00ms | -77.00ms 🚀 |
| dist/NotFound.98e75551.js | 468.00b | +0.00b | 341.00ms | -78.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 247.00ms | -29.00ms 🚀 |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 3.19m | -10.37s 🚀 |
| Cached | 5.85s | +109.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/ak-editor.4341ac4e.js | 2.69mb | +9.00b ⚠️ | 1.80m | -6.25s 🚀 |
| dist/popup.fe67861a.js | 218.11kb | +0.00b | 1.08m | +3.75s ⚠️ |
| dist/EmojiPickerComponent.20f6055b.js | 98.66kb | +0.00b | 14.17s | -54.44s 🚀 |
| dist/media-viewer.a42ca57a.js | 71.07kb | +0.00b | 19.51s | -1.58m 🚀 |
| dist/card.e4d8099d.js | 61.60kb | +0.00b | 17.84s | +3.77s ⚠️ |
| dist/card.f4e9ff08.js | 61.28kb | +0.00b | 1.80m | +45.78s ⚠️ |
| dist/component.df0df6bb.js | 56.73kb | +0.00b | 7.34s | -2.03s 🚀 |
| dist/Modal.c635528d.js | 40.56kb | +0.00b | 3.85s | -1.44s 🚀 |
| dist/component.a8512b92.js | 28.98kb | +0.00b | 6.99s | -2.11s 🚀 |
| dist/DatePicker.78d22ba2.js | 21.54kb | +0.00b | 11.91s | +980.00ms ⚠️ |
| dist/EmojiPickerComponent.38775e63.js | 18.33kb | +0.00b | 11.91s | -53.66s 🚀 |
| dist/js.baeda348.js | 16.46kb | +0.00b | 7.23s | -2.10s 🚀 |
| dist/browser.24942986.js | 16.27kb | +0.00b | 1.80m | +45.78s ⚠️ |
| dist/dropzone.35a61bda.js | 14.89kb | +0.00b | 1.80m | +45.78s ⚠️ |
| dist/ui.9e4d74b0.js | 14.45kb | +0.00b | 17.68s | +3.62s ⚠️ |
| dist/workerHasher.ebf0b386.js | 11.83kb | +0.00b | 14.79s | +2.94s ⚠️ |
| dist/EmojiTypeAheadComponent.899f7d25.js | 9.20kb | +0.00b | 14.79s | +2.94s ⚠️ |
| dist/card.45c3eb56.js | 6.23kb | +0.00b | 17.68s | +3.62s ⚠️ |
| dist/media-viewer.2db4dc6f.js | 4.18kb | +0.00b | 20.19s | +4.93s ⚠️ |
| dist/EmojiPickerComponent.dc1e6b3c.js | 3.92kb | +0.00b | 11.91s | -56.69s 🚀 |
| dist/16.cb2f1a3a.js | 3.72kb | +0.00b | 3.85s | -1.28s 🚀 |
| dist/Modal.1111ef83.js | 3.50kb | +0.00b | 6.95s | +1.78s ⚠️ |
| dist/png-chunks-extract.c0f57e67.js | 3.49kb | +0.00b | 7.25s | -2.08s 🚀 |
| dist/dropzone.79e05ece.js | 3.44kb | +0.00b | 1.01m | +4.17s ⚠️ |
| dist/clipboard.f1852e5e.js | 2.98kb | +0.00b | 1.06m | +6.89s ⚠️ |
| dist/16.c7be7376.js | 2.54kb | +0.00b | 7.23s | -2.10s 🚀 |
| dist/EmojiUploadComponent.4946a1c6.js | 2.50kb | +0.00b | 14.79s | +2.94s ⚠️ |
| dist/ResourcedEmojiComponent.0979b683.js | 2.24kb | +0.00b | 6.96s | -2.08s 🚀 |
| dist/card.444c6665.js | 2.19kb | +0.00b | 20.19s | +4.93s ⚠️ |
| dist/16.bb063a1f.js | 1.83kb | +0.00b | 7.57s | -2.04s 🚀 |
| dist/media-picker-analytics-error-boundary.187d26a0.js | 1.82kb | +0.00b | 1.01m | +4.17s ⚠️ |
| dist/date.56d1aa9e.js | 1.81kb | +0.00b | 7.91s | -1.86s 🚀 |
| dist/16.b1b0372d.js | 1.77kb | +0.00b | 8.48s | -2.32s 🚀 |
| dist/images.16b3af40.js | 1.75kb | +0.00b | 8.10s | -1.75s 🚀 |
| dist/feedback.b59078b6.js | 1.72kb | +0.00b | 58.04s | +47.11s ⚠️ |
| dist/workerHasher.9975765a.js | 1.65kb | +0.00b | 14.79s | +2.94s ⚠️ |
| dist/list-number.d2097959.js | 1.54kb | +0.00b | 8.28s | -1.66s 🚀 |
| dist/status.f61d9ac5.js | 1.54kb | +0.00b | 8.52s | -1.54s 🚀 |
| dist/16.450f3a0d.js | 1.49kb | +0.00b | 7.48s | -1.98s 🚀 |
| dist/code.4289d7dd.js | 1.46kb | +0.00b | 7.91s | -1.84s 🚀 |
| dist/16.2b801dd1.js | 1.44kb | +0.00b | 8.48s | -2.32s 🚀 |
| dist/16.fe51c023.js | 1.44kb | +0.00b | 8.52s | -948.00ms 🚀 |
| dist/16.7d4b2402.js | 1.42kb | +0.00b | 7.78s | -1.95s 🚀 |
| dist/link.36b44af7.js | 1.38kb | +0.00b | 8.22s | -5.50s 🚀 |
| dist/heading6.675c185b.js | 1.38kb | +0.00b | 58.04s | +47.11s ⚠️ |
| dist/16.6b0e338c.js | 1.38kb | +0.00b | 7.70s | -1.96s 🚀 |
| dist/heading3.4e88bcd6.js | 1.37kb | +0.00b | 8.56s | -1.62s 🚀 |
| dist/16.e32aa49c.js | 1.34kb | +0.00b | 3.85s | -1.28s 🚀 |
| dist/16.50f9c790.js | 1.33kb | +0.00b | 7.69s | -1.93s 🚀 |
| dist/emoji.3654aec7.js | 1.31kb | +0.00b | 8.10s | -1.75s 🚀 |
| dist/media-card-analytics-error-boundary.849f68c6.js | 1.31kb | +0.00b | 35.11s | +19.84s ⚠️ |
| dist/16.81746ce1.js | 1.29kb | +0.00b | 7.70s | -1.92s 🚀 |
| dist/16.d1b58d7c.js | 1.29kb | +0.00b | 7.91s | -1.82s 🚀 |
| dist/16.1dbb5df4.js | 1.28kb | +0.00b | 3.85s | -1.28s 🚀 |
| dist/Modal.6509cbe3.js | 1.26kb | +0.00b | 6.96s | -2.37s 🚀 |
| dist/heading5.e338f1e8.js | 1.25kb | +0.00b | 58.04s | +47.11s ⚠️ |
| dist/16.ceac57cd.js | 1.25kb | +0.00b | 7.49s | -2.03s 🚀 |
| dist/expand.4d883914.js | 1.24kb | +0.00b | 8.56s | -2.37s 🚀 |
| dist/16.0efadc10.js | 1.23kb | +0.00b | 8.51s | -2.39s 🚀 |
| dist/16.35be54a4.js | 1.23kb | +0.00b | 7.48s | -1.98s 🚀 |
| dist/16.6be9bbc8.js | 1.23kb | +0.00b | 7.49s | -1.98s 🚀 |
| dist/16.9015e8cf.js | 1.23kb | +0.00b | 7.25s | -2.60s 🚀 |
| dist/16.234ee133.js | 1.23kb | +0.00b | 3.85s | -5.19s 🚀 |
| dist/heading2.89a8e126.js | 1.19kb | +0.00b | 8.56s | -1.62s 🚀 |
| dist/16.7ca41ffe.js | 1.16kb | +0.00b | 7.70s | -1.92s 🚀 |
| dist/mention.ae379620.js | 1.15kb | +0.00b | 8.28s | -1.65s 🚀 |
| dist/heading4.b14d5ea4.js | 1.14kb | +0.00b | 58.04s | +47.86s ⚠️ |
| dist/layout.d9df99ef.js | 1.13kb | +0.00b | 8.10s | -1.75s 🚀 |
| dist/divider.4476c3de.js | 1.11kb | +0.00b | 8.10s | -1.75s 🚀 |
| dist/quote.4733acb8.js | 1.10kb | +0.00b | 8.43s | -1.63s 🚀 |
| dist/action.b9409056.js | 1.08kb | +0.00b | 7.91s | -1.84s 🚀 |
| dist/decision.77ee0aaa.js | 1.07kb | +0.00b | 7.97s | -1.88s 🚀 |
| dist/panel-warning.e55309d2.js | 1.06kb | +0.00b | 8.43s | -1.63s 🚀 |
| dist/list.cf500a90.js | 1.03kb | +0.00b | 8.28s | -1.66s 🚀 |
| dist/heading1.165705a7.js | 1.03kb | +0.00b | 8.56s | -1.62s 🚀 |
| dist/media-card-analytics-error-boundary.b3d7f02c.js | 995.00b | +0.00b | 17.68s | +3.62s ⚠️ |
| dist/media-card-analytics-error-boundary.a014425d.js | 995.00b | +0.00b | 1.80m | +45.78s ⚠️ |
| dist/panel-error.364fbf3c.js | 987.00b | +0.00b | 8.28s | -1.65s 🚀 |
| dist/panel.e4fe05d1.js | 978.00b | +0.00b | 8.43s | -1.63s 🚀 |
| dist/table.fd1280b6.js | 970.00b | +0.00b | 8.56s | -1.62s 🚀 |
| dist/panel-success.66723ab0.js | 926.00b | +0.00b | 8.43s | -1.63s 🚀 |
| dist/panel-note.a8b696c4.js | 922.00b | +0.00b | 8.38s | -1.55s 🚀 |
| dist/media-viewer-analytics-error-boundary.ee954f0e.js | 892.00b | +0.00b | 20.19s | +4.93s ⚠️ |
| dist/simpleHasher.dd9d4306.js | 609.00b | +0.00b | 15.88s | +1.82s ⚠️ |
| dist/index.html | 127.00b | +0.00b | 1.09s | -5.26s 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/ak-editor.4341ac4e.js | 2.69mb | +0.00b | 597.00ms | -120.00ms 🚀 |
| dist/pdfRenderer.74642224.js | 1.11mb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/editorView.918fe31c.js | 586.93kb | +0.00b | 594.00ms | -111.00ms 🚀 |
| dist/popup.fe67861a.js | 218.11kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/EmojiPickerComponent.20f6055b.js | 98.66kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/Toolbar.a68787f4.js | 93.03kb | +0.00b | 594.00ms | -111.00ms 🚀 |
| dist/media-viewer.a42ca57a.js | 71.07kb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/card.e4d8099d.js | 61.60kb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/card.f4e9ff08.js | 61.28kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/component.df0df6bb.js | 56.73kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/Modal.c635528d.js | 40.56kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/component.a8512b92.js | 28.98kb | +0.00b | 596.00ms | -121.00ms 🚀 |
| dist/DatePicker.78d22ba2.js | 21.54kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/EmojiPickerComponent.38775e63.js | 18.33kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/js.baeda348.js | 16.46kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/browser.24942986.js | 16.27kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/dropzone.35a61bda.js | 14.89kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/smartMediaEditor.ccb41b38.js | 14.62kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/ui.9e4d74b0.js | 14.45kb | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/workerHasher.ebf0b386.js | 11.83kb | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/EmojiTypeAheadComponent.899f7d25.js | 9.20kb | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/card.45c3eb56.js | 6.23kb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/media-viewer.2db4dc6f.js | 4.18kb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/EmojiPickerComponent.dc1e6b3c.js | 3.92kb | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/16.cb2f1a3a.js | 3.72kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/Modal.1111ef83.js | 3.50kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/png-chunks-extract.c0f57e67.js | 3.49kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/ak-editor.e6be06da.css | 3.46kb | +0.00b | 594.00ms | -111.00ms 🚀 |
| dist/dropzone.79e05ece.js | 3.44kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/clipboard.f1852e5e.js | 2.98kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/16.c7be7376.js | 2.54kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/EmojiUploadComponent.4946a1c6.js | 2.50kb | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/ResourcedEmojiComponent.0979b683.js | 2.24kb | +0.00b | 596.00ms | -121.00ms 🚀 |
| dist/card.444c6665.js | 2.19kb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/16.bb063a1f.js | 1.83kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.187d26a0.js | 1.82kb | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/date.56d1aa9e.js | 1.81kb | +0.00b | 596.00ms | -119.00ms 🚀 |
| dist/16.b1b0372d.js | 1.77kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/images.16b3af40.js | 1.75kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/feedback.b59078b6.js | 1.72kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/workerHasher.9975765a.js | 1.65kb | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/list-number.d2097959.js | 1.54kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/status.f61d9ac5.js | 1.54kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/16.450f3a0d.js | 1.49kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/code.4289d7dd.js | 1.46kb | +0.00b | 596.00ms | -119.00ms 🚀 |
| dist/16.2b801dd1.js | 1.44kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.fe51c023.js | 1.44kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.7d4b2402.js | 1.42kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/link.36b44af7.js | 1.38kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/heading6.675c185b.js | 1.38kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/16.6b0e338c.js | 1.38kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/heading3.4e88bcd6.js | 1.37kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/16.e32aa49c.js | 1.34kb | +0.00b | 596.00ms | -121.00ms 🚀 |
| dist/16.50f9c790.js | 1.33kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/emoji.3654aec7.js | 1.31kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/media-card-analytics-error-boundary.849f68c6.js | 1.31kb | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/16.81746ce1.js | 1.29kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.d1b58d7c.js | 1.29kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/16.1dbb5df4.js | 1.28kb | +0.00b | 596.00ms | -121.00ms 🚀 |
| dist/Modal.6509cbe3.js | 1.26kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/heading5.e338f1e8.js | 1.25kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/16.ceac57cd.js | 1.25kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/expand.4d883914.js | 1.24kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/16.0efadc10.js | 1.23kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.35be54a4.js | 1.23kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.6be9bbc8.js | 1.23kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.9015e8cf.js | 1.23kb | +0.00b | 597.00ms | -119.00ms 🚀 |
| dist/16.234ee133.js | 1.23kb | +0.00b | 596.00ms | -121.00ms 🚀 |
| dist/heading2.89a8e126.js | 1.19kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/16.7ca41ffe.js | 1.16kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/mention.ae379620.js | 1.15kb | +0.00b | 595.00ms | -121.00ms 🚀 |
| dist/heading4.b14d5ea4.js | 1.14kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/layout.d9df99ef.js | 1.13kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/divider.4476c3de.js | 1.11kb | +0.00b | 596.00ms | -119.00ms 🚀 |
| dist/quote.4733acb8.js | 1.10kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/action.b9409056.js | 1.08kb | +0.00b | 596.00ms | -120.00ms 🚀 |
| dist/decision.77ee0aaa.js | 1.07kb | +0.00b | 596.00ms | -119.00ms 🚀 |
| dist/panel-warning.e55309d2.js | 1.06kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/list.cf500a90.js | 1.03kb | +0.00b | 595.00ms | -121.00ms 🚀 |
| dist/heading1.165705a7.js | 1.03kb | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/media-card-analytics-error-boundary.b3d7f02c.js | 995.00b | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/media-card-analytics-error-boundary.a014425d.js | 995.00b | +0.00b | 595.00ms | -119.00ms 🚀 |
| dist/panel-error.364fbf3c.js | 987.00b | +0.00b | 595.00ms | -121.00ms 🚀 |
| dist/panel.e4fe05d1.js | 978.00b | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/table.fd1280b6.js | 970.00b | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/panel-success.66723ab0.js | 926.00b | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/panel-note.a8b696c4.js | 922.00b | +0.00b | 595.00ms | -120.00ms 🚀 |
| dist/media-viewer-analytics-error-boundary.ee954f0e.js | 892.00b | +0.00b | 594.00ms | -120.00ms 🚀 |
| dist/EmojiPickerComponent.8bf91f55.js | 616.00b | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/simpleHasher.dd9d4306.js | 609.00b | +0.00b | 594.00ms | -121.00ms 🚀 |
| dist/index.html | 127.00b | +0.00b | 598.00ms | -121.00ms 🚀 |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/268724680913846792)**

**Comment by devongovett** [comment]
We don't really have a good test case for this yet in core since none of the builtin plugins need this yet.

**Comment by mischnic** [comment]
I guess we should have at least one basic functional test?

**Comment by devongovett** [comment]
Yeah... wasn't sure how to do that since none of the core plugins need to do this. Maybe a custom plugin in the test I guess.

**Pull request merged**